.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
- [ ] OPTIONAL
To enhance a more professional repository kindly ensure to add the `.vscode` folder in the `.gitignore` file.

**Your current project repository file structure:**
![image](https://user-images.githubusercontent.com/12745474/129293893-65f80d2c-845f-418f-94dd-a80339b62eb2.png)